src/FM.LiveSwitch.Mux/Recording.cs

```diff
@@ -137,7 +137,7 @@ public VideoEvent[] VideoEvents
                 videoEvents.Add(new VideoEvent
                 {
                     Type = VideoEventType.Remove,
-                    Timestamp = VideoStopTimestamp.Value,
+                    Timestamp = VideoStopTimestamp ?? DateTime.MinValue,
```

Comment:
Can we get rid of the use of `DateTime.MinValue` now that we have null checks? This line and the change above it (same) can be reverted?